package.json

```diff
@@ -96,5 +98,18 @@
     "type": "opencollective",
     "url": "https://opencollective.com/parse-server",
     "logo": "https://opencollective.com/parse-server/logo.txt?reverse=true&variant=binary"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "{src,spec}/**/*.js": [
+      "prettier --write",
+      "flow",
+      "eslint --cache ./",
+      "git add"
```

Comment:
@flovilmart a bad side effect of this change is that if you commit with unstaged changes, they'll actually get committed.

Reply:
No they don’t only the list of files that are staged are passed as an argument.

Husky passes the list of files to the commandsz

Reply:
In my case, I staged part of a file, but it would end up committing all the changes in the file, not just the parts I staged.

Reply:
You’re right, still, this feels like a corner case

Reply:
I can live with it